mem/mem.c

```diff
@@ -1406,6 +1411,7 @@ static void destroy_zone(void *arg)
             }
 
         mspace_free(root.m, arg);
+        t_zone = NULL;
```

Comment:
This is needless as the thread is already exiting.

Reply:
yep, so was the initial = NULL. Thanks.